src/commands/init/index.ts

```diff
@@ -79,4 +81,48 @@ export default class Init extends Command {
 
     ux.action.stop();
   }
+
+  private async getStartAppMessage(): Promise<string> {
+    const docsUrl = 'https://docs.demo.saas.apptoku.com/';
+    const landingUrl = 'https://www.apptension.com/saas-boilerplate';
+
+    return `
+——————————————————————————————————————————————————————————————————————————————————————————————————————
+Initialization completed! 🚀 To start the application, please refer to the following commands:
+
+| \u001B[1mStart backend:\u001B[0m
+|
+| \u001B[34mnx run core:docker-compose:up\u001B[0m
+|
+| or a shorter version:
+|
+| \u001B[34mmake up\u001B[0m
+|
+| This will run docker containers for all the backend services in the detached mode.
+|
+| Backend is running on \u001B[34mhttp://localhost:5001\u001B[0m.
+|
+| Admin Panel is running on \u001B[34mhttp://admin.localhost:5001\u001B[0m.
+|
+| Workers trigger server is running on \u001B[34mhttp://localhost:3005\u001B[0m.
```

Comment:
Not sure if we need this because this is just a local listener and it's not returning anything to browser. I would remove this line

Reply:
changes applied